client/client-multi/client-c/thirdparty/mbedtls/pkparse.c

```diff
@@ -80,7 +80,6 @@ int mbedtls_pk_load_file( const char *path, unsigned char **buf, size_t *n )
 
     if( ( f = fopen( path, "rb" ) ) == NULL )
         return( MBEDTLS_ERR_PK_FILE_IO_ERROR );
-
```

Comment:
Return back this line.
It is not recommended to edit thirdparty files as far is this is not critical.